woocommerce/woocommerce-ios
# [Order Creation - Customer Selection] Show selector when addresses are empty

Closes: #


## Description

With this PR we fix a bug that was causing showing the new customer form instead of the customer selector when a product is added. 
Before we were just checking if the addresses were nil to show the selector, but this is not enough: we have to check if they're empty as well. The reason behind this is that one of the latest changes in how we fill data to the order was resetting the addresses making them empty.

## Testing instructions


1. Go to Orders
2. Tap on + to create a new order
3. Add a new product
4. Tap on Add Customer Selector
5. See that the customer selector is shown, not the customer form

## Screenshots


### Before

https://github.com/woocommerce/woocommerce-ios/assets/1864060/8f0165dd-1598-4e48-b74c-f2e7293c5a2d

### After

https://github.com/woocommerce/woocommerce-ios/assets/1864060/23e717ae-d408-4fdb-9455-d7df5ff6f87d


---
